### PR TITLE
docs: add `logging.InjectFields` usage description

### LINKED ISSUE #521: Update v2 readme for how to use `logging.ExtractFields`
I saw [v2 changelog](https://github.com/grpc-ecosystem/go-grpc-middleware/blob/v2/CHANGELOG.md#changed) says:

> `tags` removed. Use `logging.ExtractFields` to read logging fields from logging interceptor for your local request logger. Use `logging.InjectFields` to inject custom fields to logging interceptor to client context or interceptor before logging interceptor.

In the [v2 readme](https://github.com/grpc-ecosystem/go-grpc-middleware/tree/v2#middleware) demo, it still uses old `tags`:

```go
import "github.com/grpc-ecosystem/go-grpc-middleware/v2"

myServer := grpc.NewServer(
    grpc.ChainStreamInterceptor(
        tags.StreamServerInterceptor(),
        opentracing.StreamServerInterceptor(),
        prometheus.StreamServerInterceptor,
        zap.StreamServerInterceptor(zapLogger),
        auth.StreamServerInterceptor(myAuthFunction),
        recovery.StreamServerInterceptor(),
    ),
    grpc.ChainUnaryInterceptor(
        tags.UnaryServerInterceptor(),
        opentracing.UnaryServerInterceptor(),
        prometheus.UnaryServerInterceptor,
        zap.UnaryServerInterceptor(zapLogger),
        auth.UnaryServerInterceptor(myAuthFunction),
        recovery.UnaryServerInterceptor(),
    ),
)
```

It would be great to update readme for how to use `logging.ExtractFields`. Thanks! 😃

### PR DESCRIPTION
**Pull request by aimuz**
Fixed #521

### CODE REVIEW

**Review by johanbrandhorst — COMMENTED** [review]
I'm confused by this change. Did we remove the `tags` interceptor? If so, we should just remove it from the list, not keep it strikethrough.

**Comment by aimuz** [comment]
The implementation of tags has been removed in v2.

File: README.md [hunk 1]
```diff
@@ -58,7 +56,8 @@ myServer := grpc.NewServer(
 
 #### Logging
 
-   * [`tags`](interceptors/tags) - a library that adds a `Tag` map to context, with data populated from request body
+   * ~~[`tags`](interceptors/tags) - a library that adds a `Tag` map to context, with data populated from request body.~~ replace `tags.Extract().Set()` to `logging.InjectFields()`
```
Comment (johanbrandhorst) [inline README.md:59]: Lets just remove this line

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Thanks for the clarification, lets just remove this.

**Review by johanbrandhorst — APPROVED** [review]

**Comment by johanbrandhorst** [comment]
Thanks for your contribution!

**Pull request merged**